src/BackgroundTasks/Implementation/TaskManager/AsyncTaskManager.php

```diff
@@ -55,9 +55,14 @@ public function run(Bucket $bucket): void
             )
             : '';
         try {
-            $soap_client->call(self::CMD_START_WORKER, array(
+            $result = $soap_client->call(self::CMD_START_WORKER, array(
                 $session_id . '::' . $client_id,
             ));
+            if ($result === false) {
+                $DIC->logger()->root()->info("[BT] Calling Webserver returned false, fallback to sync version");
```

Comment:
Shouldn't we use `warning` or `error` in this case?

Reply:
This would create many entries, if soap is not active. @chfsx may decide this.

Reply:
I suggest to use the component logger for BackgroundTask instead of root and then use warning.